MapboxNavigation/EndOfRouteViewController.swift

```diff
@@ -69,137 +87,58 @@ class EndOfRouteViewController: UIViewController {
 
-    private func styleCommentView() {
-        self.commentView.layer.cornerRadius = 6.0
-        self.commentView.layer.borderColor = UIColor.lightGray.cgColor
-        self.commentView.layer.borderWidth = 1.0
-        self.commentView.textContainerInset = UIEdgeInsets(top: 10.0, left: 10.0, bottom: 10.0, right: 10.0)
-    }
-    
-    fileprivate func dismissView() {
-        let dismissal: () -> Void = { self.dismissHandler?(self.rating, self.comment) }
-        guard self.commentView.isFirstResponder else { return _ = dismissal() }
-        self.commentView.resignFirstResponder()
-        let fireTime = DispatchTime.now() + 0.3 // Not ideal, but works for now
-        DispatchQueue.main.asyncAfter(deadline: fireTime, execute: dismissal)
-    }
-    
-    private func showComments(animated: Bool = true) {
-        self.showCommentView.isActive = true
-        self.hideCommentView.isActive = false
-        self.ratingCommentsSpacing.constant = ConstraintSpacing.closer.rawValue
-        preferredContentSize.height = self.height(for: .commentShowing)
-
-        let animate = {
-            self.view.layoutIfNeeded()
-            self.commentViewContainer.alpha = 1.0
-            self.labelContainer.alpha = 0.0
-        }
-        
-        let completion: (Bool) -> Void = { _ in self.labelContainer.isHidden = true }
-        let noAnimate = { animate(); completion(true) }
-        if animated {
-            UIView.animate(withDuration: 0.3, animations: animate, completion: nil)
-        } else {
-            noAnimate()
-        }
-    }
-    
-    private func hideComments(animated: Bool = true) {
-        self.labelContainer.isHidden = false
-        self.showCommentView.isActive = false
-        self.hideCommentView.isActive = true
-        self.ratingCommentsSpacing.constant = ConstraintSpacing.further.rawValue
-        preferredContentSize.height = self.height(for: .normal)
-        
-        let animate = {
-            self.view.layoutIfNeeded()
-            self.commentViewContainer.alpha = 0.0
-            self.labelContainer.alpha = 1.0
-        }
-        
-        let completion: (Bool) -> Void = { _ in self.commentViewContainer.isHidden = true }
-        let noAnimation = { animate(); completion(true) }
-        if animated {
-            UIView.animate(withDuration: 0.3, animations: animate, completion: nil)
-        } else { noAnimation() }
-    }
-    
     private func height(for height: ContainerHeight) -> CGFloat {
         let window = UIApplication.shared.keyWindow
         let bottomMargin = window!.safeArea.bottom
         return height.rawValue + bottomMargin
     }
     
     private func updateInterface() {
-        guard let name = destination?.name?.nonEmptyString else { return self.styleForUnnamedDestination() }
-        self.primary.text = name
-    }
-
-    private func clearInterface() {
-        self.primary.text = nil
-        self.stars.rating = 0
-    }
-    
-    private func styleForUnnamedDestination() {
```

Comment:
I inlined this because it was short, and easier to understand the parallel logic of the "else" case.

For example, we now set `self.staticYouHaveArrived.alpha` back to `1.0` in the case that a new destination has a name after a previous one doesn't.